docset/winserver2022-ps/defender/Remove-MpPreference.md

```diff
@@ -45,6 +71,51 @@ This command will exclude only that specific file app.exe in that specific folde
 
 ## PARAMETERS
 
+### -AllowDatagramProcessingOnWinServer
+Removes whether to disable inspection of UDP connections on Windows Server.
```

Comment:
Is _Removes_ the correct word here or maybe _Determines_? the same for the other parameters...